hyperspy/misc/array_tools.py

```diff
@@ -301,6 +308,9 @@ def _linear_bin(dat, scale, crop=True):
         # The new dimension size is old_size/step, this is rounded down normally
         # but if crop is switched off it is rounded up to the nearest whole
         # number.
+        if not np.issubtype(s, float):
```

Comment:
So Travis is failing because of: 
```
FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
```
Does that mean converting to np.issubdtype(s, np.float64) or np.issubdtype(s, np.dtype(float).type)?